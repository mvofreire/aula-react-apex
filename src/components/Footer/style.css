.footer{
  position: absolute;
  bottom: 0px;
  width: 100%;
  background: #f0f0f0;
  padding: 20px;
}