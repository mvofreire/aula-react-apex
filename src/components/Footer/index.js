import React from "react";
import "./style.css"
function Footer(){

  return (
    <div className="footer">
      Powered By @apexEnsino
    </div>
  )
}

export default Footer