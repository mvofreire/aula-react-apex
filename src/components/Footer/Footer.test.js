import { render } from '@testing-library/react';
import Footer from '.';

describe('Footer', () => {
  test('Should render in screen', () => {
    const { container } = render(<Footer />)
    expect(container).toBeInTheDocument()
  })

  test('Should match with snapshot', () => {
    const { container } = render(<Footer />)
    expect(container).toMatchSnapshot('default')
  })
})