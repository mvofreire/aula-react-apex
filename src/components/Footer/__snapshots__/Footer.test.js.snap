// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer Should match with snapshot: default 1`] = `
<div>
  <div
    class="footer"
  >
    Powered By @apexEnsino
  </div>
</div>
`;
